test/javascript/components/offcanvas.js

```diff
@@ -104,7 +104,7 @@ describe('Off Canvas', function() {
           plugin.$element.should.have.class('is-open');
           $('body').should.have.class('is-off-canvas-open');
           done();
-        }, 1);
+        }, 30);
```

Comment:
See https://github.com/zurb/foundation-sites/pull/10943#discussion_r168351520

I think we need a timeout here because the opened.zf.offcanvas event is triggered before the class and everything is set, which is not a good thing (because the offcanvas is not fully opened).

We should resolve that instead of putting a workarounds in the tests.

Reply:
Definitely.